import-export-cli/cmd/addEnv.go

```diff
@@ -54,9 +53,10 @@ const addEnvCmdExamples = utils.ProjectName + ` ` + addEnvCmdLiteral + ` -e prod
 --registration https://idp.com:9443 \
 --token https://gw.com:8243/token
 
-NOTE: The flags --environment (-e) and --token are mandatory
-You can either provide only the 2 flags --apim and --token, or all the other 5 flags (--registration --publisher --devportal --admin --token) without providing --apim flag.
-If you are omitting any of --registration --publisher --devportal --admin flags, you need to specify --apim flag with the API Manager endpoint.`
+NOTE: The flag --environment (-e) is mandatory.
+You can either provide only the 1 flag --apim , or all the other 4 flags (--registration --publisher --devportal --admin --token) without providing --apim flag.
```

Comment:
Let's change as below.
"You can either provide only the flag --apim , or all the other 4 flags "